src/app/server/Server.h

```diff
@@ -214,10 +218,15 @@ struct CommonCaseDeviceServerInitParams : public ServerInitParams
         // PersistentStorageDelegate "software-based" operational key access injection
         if (this->operationalKeystore == nullptr)
         {
+#if CHIP_CRYPTO_PSA
+            ReturnErrorOnFailure(sPSAOperationalKeystore.Init(this->persistentStorageDelegate));
```

Comment:
This will fail to work for any client using a custom ServerInitParams. I guess that's OK, but bear in mind some existing products will then miss the migration.

Also, failing on migration means that the product will error-out Server::Init, which *may lead to bootlooping*. It is extremely risky to do "database migration" in a way that errors are not passively logged/ignored. It means that if anything arises which was not seen in the field, products that upgrade to this new code can get bricked. I would recommend leaving it to individual products to opt-in to this migration, or alternatively, make sure that it cannot cause a failure of Server Init.